tests/pipelines/stable_diffusion/test_stable_diffusion_adapter.py

```diff
@@ -404,14 +404,11 @@ def test_inference_batch_single_identical(
         if test_mean_pixel_difference:
             assert_mean_pixel_difference(output_batch[0][0], output[0][0])
 
-    # We do not support saving pipelines with multiple adapters. The multiple adapters should be saved as their
-    # own independent pipelines
-
     def test_save_load_local(self):
-        ...
+        super().test_save_load_local()
 
     def test_save_load_optional_components(self):
-        ...
+        super().test_save_load_optional_components()
```

Comment:
Actually if it works we can just delete the tests here, then it will call the automatically :-) The only reason we previously added them here was so that we can skip the tests

```suggestion
```

Reply:
Ah, ok I see :)